libpod/oci.go

```diff
@@ -789,9 +789,20 @@ func (r *OCIRuntime) execContainer(c *Container, cmd, capAdd, env []string, tty
 	logrus.Debugf("Starting runtime %s with following arguments: %v", r.path, args)
 
 	execCmd := exec.Command(r.path, args...)
-	execCmd.Stdout = os.Stdout
-	execCmd.Stderr = os.Stderr
-	execCmd.Stdin = os.Stdin
+	execCmd.Stdout = streams.OutputStream
+	execCmd.Stderr = streams.ErrorStream
+	execCmd.Stdin = streams.InputStream
```

Comment:
Why not just do these calls below 

if streams.AttachError {
          execCmd.Stdin = streams.InputStream

Reply:
ah yeah that's better